src/lib.rs

```diff
@@ -264,9 +255,7 @@ pub async fn bootstap(mut shutdown: broadcast::Receiver<()>, config: Config) ->
 
     #[cfg(not(feature = "multitenant"))]
     let app = Router::new()
-        .route("/health", get(handlers::health::handler).layer(
-            axum::middleware::from_fn_with_state(state_arc.clone(), rate_limit_middleware),
-        ))
+        .route("/health", get(handlers::health::handler))
```

Comment:
Removes rate limits on `/health` as 1) they aren't needed and 2) they fail to resolve the `Forwarded-For` header anyway since it's called internally